ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```diff
@@ -100,6 +100,72 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-aws-operator-periodic
+  interval: 72h
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e-periodic
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
+- as: e2e-vsphere-operator-periodic
+  interval: 72h
+  steps:
+    cluster_profile: vsphere
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e-periodic
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: openshift-e2e-vsphere
+- as: e2e-azure-operator-periodic
+  interval: 72h
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e-periodic
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-azure
+- as: e2e-gcp-operator-periodic
+  interval: 72h
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make -C /go/src/github.com/openshift/cluster-api-actuator-pkg
```

Comment:
why is this different?

Reply:
As far as i can say - this will run `test-e2e` make target.

I would expect to see `JUNIT_DIR=${ARTIFACT_DIR} make test-e2e-periodic `